db/migrate/20170816193809_remove_string_value_from_calculator_constant.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveStringValueFromCalculatorConstant < ActiveRecord::Migration
+  def change
+    remove_column :calculator_constants, :string_value
```

Comment:
I think there are 22 calculator constants when this migration is run. I looked at the `pg_attribute` table entries for `calculator_constants, and the `string_value` column is not listed. I  assume that were it present (even disabled) it would be in that list.

![screen shot 2017-08-23 at 11 13 42 am](https://user-images.githubusercontent.com/4104431/29623401-39e99a82-87f4-11e7-8d27-e50f078c91e3.png)